apps/ledger-live-desktop/tests/utils/serve-dummy-app.ts

```diff
@@ -2,15 +2,19 @@ import handler from "serve-handler";
 import http from "http";
 import path from "path";
 
+let dummyAppPath: string;
+
 export const server = http.createServer((request, response) => {
   // You pass two more arguments for config and middleware
   // More details here: https://github.com/vercel/serve-handler#options
-  return handler(request, response, {
-    public: path.resolve(__dirname, "dummy-live-app/build"),
+  handler(request, response, {
+    public: path.resolve(__dirname, dummyAppPath),
   });
 });
 
-export const start = (port = 0): Promise<number> => {
+export const start = (appPath: string, port = 0): Promise<number> => {
```

Comment:
```suggestion
export const start = (appPath: string, port: number = 0): Promise<number> => {
```

Reply:
It says it doesn't need it as it can determine the type by the `0` 
![image](https://user-images.githubusercontent.com/87659842/231423430-34d9f869-78ed-4f2f-91f7-b5ce4d768e95.png)